sw/host/tests/manuf/manuf_cp_ast_test_execution/src/main.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::time::Duration;
+
+use anyhow::{Context, bail};
+use structopt::StructOpt;
+
+use opentitanlib::io::jtag::{JtagTap, JtagParams, RiscvReg, RiscvCsr};
+use opentitanlib::test_utils::init::InitializeTest;
+use opentitanlib::test_utils::load_sram_program::{SramProgramParams, ExecutionMode, ExecutionResult};
+use opentitanlib::uart::console::UartConsole;
+
+#[derive(Debug, StructOpt)]
+struct Opts {
+    #[structopt(flatten)]
+    init: InitializeTest,
+    #[structopt(flatten)]
+    jtag: JtagParams,
+    #[structopt(flatten)]
+    sram_program: SramProgramParams,
+    #[structopt(
+        long, parse(try_from_str=humantime::parse_duration),
+        default_value = "600s",
+        help = "Console receive timeout",
+    )]
+    timeout: Duration,
+}
+
+fn main() -> anyhow::Result<()> {
+    let opts = Opts::from_args();
+    opts.init.init_logging();
+    let transport = opts.init.init_target()?;
+    let uart = transport.uart("console")?;
+    uart.set_flow_control(true)?;
```

Comment:
no need to setup flow control here, the device side code isn't sending any ujson commands, nor does it turn on flow control itself